membership_policy.md

```diff
@@ -128,6 +130,82 @@ The following apply to the part of project for which one would be an approver in
   * Mentor contributors and reviewers
   * May approve contributions for acceptance
 
+## Special Interest Group (SIG) Chair
+
+The KubeVirt project is organized primarily into SIGs, each with a common purpose of advancing the project with respect to a specific topic, such as Networking or Storage. A SIG Chair is required to help guide and coordinate the SIG. 
+
+### Requirements
+
+* Active reviewer for SIG matters for at least 3 months
```

Comment:
This requirement is a bit problematic for new SIGs...

Reply:
New SIGs will usually be created from existing areas, therefore this can still stand.
In case a new SIG is created from a totally new subject, then the candidate can show knowledge in that are from other sources. But I think this is rarely the case and we could ignore it until such an incident really happens.

Reply:
Yes, that's fine.